packages/amplify-e2e-core/src/categories/api.ts

```diff
@@ -62,6 +64,11 @@ export function addApiWithoutSchema(cwd: string, opts: Partial<AddApiOptions & {
           reject(err);
         }
       });
+
+    if (options.transformerVersion === 1) {
+      addFeatureFlag(cwd, 'graphqltransformer', 'transformerVersion', 1);
+      addFeatureFlag(cwd, 'graphqltransformer', 'useExperimentalPipelinedTransformer', false);
+    }
```

Comment:
This snippet is still copied around a bit much. Maybe add a helper function that takes `cwd` and `options`.

Reply:
added